backend/src/server/createServer.ts

```diff
@@ -41,7 +47,37 @@ const createServer = async (context: any = serverContext): Promise<ServerDef> =>
   if (!dbVersion) {
     throw new Error('Fatal: Database Version incorrect')
   }
-
+  // *********** UNICORN ADD ************************
+  // check if community alias was registered on blockchain
+  if (!isCommunityAliasExisting(CONFIG.COMMUNITY_ALIAS)) {
+    // is not registered, so we create it
+    // load first admin user for signing transaction
+    const created = new Date()
+    const adminUserRepository = getRepository(UserAdmin)
+    const adminUserResult = await adminUserRepository.find({
+      select: ['userId'],
+      order: {
+        userId: 'ASC',
+      },
+      take: 1,
+    })
```

Comment:
You need an admin user to registrate a new group? But is it really ok to take a random / first found one?

Reply:
No, but it will much more work for me to implement a proper process in admin for that. 
Also I don't know if you have already planned a kind of setup process on first start in admin. 
Currently creating an admin user isn't even possible from the UI. It is only possible with direct db access.
So my focus is to test the blockchain connector and gradido node together with apollo. We can later integrate it in an server-setup process when it exists.